bin/check-http.rb

```diff
@@ -201,7 +201,7 @@ def run
     end
 
     begin
-      Timeout.timeout(config[:timeout]) do
+      Timeout.timeout(config[:timeout] + 10) do
```

Comment:
It doesn't make any sense to have it hardcoded, but this fix needs to come upstream sooner than later. I'm seeing lots of issues with this lately, and it gives me a lot of false positives.